src/mavlink-router/endpoint.cpp

```diff
@@ -229,12 +231,17 @@ int Endpoint::read_msg(struct buffer *pbuf, int *target_sysid, int *target_compi
          * Ground Station and Flight Stack. Although it can also be a
          * corrupted message is better forward than silent drop it.
          */
-        if (!_check_crc(msg_entry)) {
-            _stat.read.crc_error++;
-            _stat.read.crc_error_bytes += expected_size;
```

Comment:
Now that I see it, these should be kept

Reply:
You're right. In the future I would like to log these metrics :+1: